runtests.py

```diff
@@ -472,5 +478,49 @@ def lcov_generate():
         print("HTML output generated under build/lcov/")
 
 
+@contextlib.contextmanager
+def working_dir(new_dir):
+    current_dir = os.getcwd()
+    try:
+        os.chdir(new_dir)
+        yield
+    finally:
+        os.chdir(current_dir)
+
+
+def run_mypy(args):
+    if args.no_build:
```

Comment:
I don't think we can run against an installed SciPy yet since we aren't yet PEP 561 compliant. I might be wrong about that though.

Reply:
This seems to be a matter of including a `py.typed` file at the top level and including there a `partial` for the parts that don't work yet I think. Or is there more to it than that?

Either way, looks to me like erroring out here is the right thing to do until the next release.